scripts/kcc

```diff
@@ -26,14 +26,24 @@ my @srcFiles = ();
 my @tempFiles = ();
 
 my $distDirectory = dirname(rel2abs($0));
+our $profile;
```

Comment:
I notice some others have crept in as well, but shouldn't all the variables in this file be declared using "my" so they're lexically scoped to this file?